Minimum_insertion_to_make_Palindrome/minimum_insertion_to_make_palindrome_optimize.py

```diff
@@ -0,0 +1,49 @@
+# Python Program to find min Insertion in a string to make it Palindrome.
+""" -----------------------------------
+Example:
+ab : No. of insertion is required 1 (bab)
+abcd : No. of insertion is required 3 (dcbabcd)
+
+Dynamic Programming based Solution.
+Time Complexity : O(n^2)
+Space Complexity : O(n^2)
+--------------------------------------- """
+# Function for Findind Min.
+def Min(a, b):
+    """
+        This Function is return minimum value B/w a and b.
+    """
+    return min(a, b)
+
```

Comment:
Remove the extra lines between functions. Only one new-line should be there between two functions.